src/pages/Quiz/index.js

```diff
@@ -7,7 +7,7 @@ import { Link } from 'react-router-dom';
 
 import classnames from 'classnames';
 
-import quizData from '../../data/quiz.json';
+import { quiz } from '../../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file